CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 1.1.2 - Internal release
```

Comment:
Minor: On room #feat-python-plugins-update, consensus seems to be that we should bump the minor (so `1.2`)

Reply:
done